enarx-autoassign

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+import enarxbot
+import random
+
+github = enarxbot.connect()
+org = github.get_organization("enarx")
+
+# Auto-assign for all PRs in the org.
+for issue in github.search_issues(f"org:enarx is:pr is:public is:open -is:draft"):
+    repo = issue.repository
+    pr = issue.as_pull_request()
+
+    if len(pr.get_review_requests()) >= 3:
+        continue
+
+    # If the PR has fewer than three reviewers requested, auto-assign.
+    # First, put Nathaniel on all code reviews as code owner.
+    npmccallum = github.get_user("npmccallum")
```

Comment:
How do you mean?

Reply:
Something like...

```python
owner_name = "npmccallum"
owner = github.get_user(owner_name)
```

I'm thinking if we want to change the owner later it might be easier dealing with a single hardcoded value

Reply:
Very good point. I've made that change.